src/library/Uno.Themes.WinUI.Markup/Theme/PipsPager.cs

```diff
@@ -0,0 +1,247 @@
+							public static ResourceValue<Brush> PointerOver => new("PipsPagerNavigationButtonBackgroundPointerOver", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "PipsPagerNavigationButtonBackgroundPressed")]
+							public static ResourceValue<Brush> Pressed => new("PipsPagerNavigationButtonBackgroundPressed", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "PipsPagerNavigationButtonBackgroundDisabled")]
+							public static ResourceValue<Brush> Disabled => new("PipsPagerNavigationButtonBackgroundDisabled", true);
+						}
+
+						public static class BorderBrush
+						{
+							[ResourceKeyDefinition(typeof(Brush), "PipsPagerNavigationButtonBorderBrush")]
+							public static ResourceValue<Brush> Default => new("PipsPagerNavigationButtonBorderBrush", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "PipsPagerNavigationButtonBorderBrushPointerOver")]
+							public static ResourceValue<Brush> PointerOver => new("PipsPagerNavigationButtonBorderBrushPointerOver", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "PipsPagerNavigationButtonBorderBrushPressed")]
+							public static ResourceValue<Brush> Pressed => new("PipsPagerNavigationButtonBorderBrushPressed", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "PipsPagerNavigationButtonBorderBrushDisabled")]
+							public static ResourceValue<Brush> Disabled => new("PipsPagerNavigationButtonBorderBrushDisabled", true);
+						}
+
+						public static class Foreground
+						{
+							[ResourceKeyDefinition(typeof(Brush), "PipsPagerNavigationButtonForeground")]
+							public static ResourceValue<Brush> Default => new("PipsPagerNavigationButtonForeground", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "PipsPagerNavigationButtonForegroundPointerOver")]
+							public static ResourceValue<Brush> PointerOver => new("PipsPagerNavigationButtonForegroundPointerOver", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "PipsPagerNavigationButtonForegroundPressed")]
+							public static ResourceValue<Brush> Pressed => new("PipsPagerNavigationButtonForegroundPressed", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "PipsPagerNavigationButtonForegroundDisabled")]
+							public static ResourceValue<Brush> Disabled => new("PipsPagerNavigationButtonForegroundDisabled", true);
+						}
+
+						public static class VisualStateEllipse
+						{
+							public static class Height
+							{
+								[ResourceKeyDefinition(typeof(double), "MaterialPipsPagerNavigationButtonVisualStateEllipseHeight")]
+								public static ResourceValue<double> Default => new("MaterialPipsPagerNavigationButtonVisualStateEllipseHeight", true);
+							}
+
+							public static class Width
+							{
+								[ResourceKeyDefinition(typeof(double), "MaterialPipsPagerNavigationButtonVisualStateEllipseWidth")]
+								public static ResourceValue<double> Default => new("MaterialPipsPagerNavigationButtonVisualStateEllipseWidth", true);
+							}
```

Comment:
No need for an extra class here, can just be properties of the VisualStateEllipse class